datalad/support/annexrepo.py

```diff
@@ -335,3 +336,103 @@ def annex_add_to_git(self, files):
             self.cmd_call_wrapper.run(cmd_list, cwd=self.path)
         else:
             self.git_add(files)
+
+    def annex_initremote(self, name, options):
+        """Creates a new special remote
+
+        Parameters:
+        -----------
+        name: str
+            name of the special remote
```

Comment:
Plus: Overriding is not a good way, while in development of basic features, because you may temporarily want to use annex- or git- methods directly from Dataset for example until you figured out how to implement current task.